fuzz_lightyear/request.py

```diff
@@ -203,6 +205,18 @@ def fuzz(self, existing_data: Dict[str, Any]) -> Dict[str, Any]:
 
         return fuzzed_input
 
+    def apply_post_fuzz_hooks(self, fuzzed_input: Dict[str, Any]) -> Dict[str, Any]:
+        """After a request is fuzzed, applies developer-supplied post-fuzz
```

Comment:
Based on our clarifying conversation in slack, we might want to disambiguate this with `after parameters for a request are fuzzed`, or make it more explicit that these hooks are applied pre-actual request.